### PR TITLE
NOISSUE - Add CoCos-AI Manager API specification

### PR DESCRIPTION
**Pull request by SammyOina**
Add openapi spec

### CODE REVIEW

File: api/manager.yml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+          example: 7d6f4dc4f7f0c1fa3dc24eddfb18bb5073ff4f62
+        description:
+          type: string
+          description: Service description.
+          example: <service_name> service
+        build_time:
+          type: string
+          description: Service build time.
+          example: 1970-01-01_00:00:00
+    ComputationReq:
```
Comment (rodneyosodo) [inline api/manager.yml:71]: This is declared but not used

**Review by rodneyosodo — CHANGES_REQUESTED** [review]

File: api/manager.yml (hunk 1 above)
Comment (rodneyosodo) [reply api/manager.yml:71]: But it is currently not used https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it

Comment (SammyOina) [reply api/manager.yml:71]: it is referred to in the spec, the bytes in the request are from the /run request arise from this

Comment (rodneyosodo) [reply api/manager.yml:71]: From the documentation, there is nothing linking the two. Something like 
```yaml
description: byte array of computation request generated from ...
```

Comment (SammyOina) [reply api/manager.yml:71]: see l130

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**